docs/roadmap.md

```diff
@@ -11,37 +11,49 @@ Here's how we intend to roll out Project Reunion:
 <table>
   <tbody>
     <tr>
-      <th>//BUILD 2020</th>
-      <th>End of 2020</th>
-      <th>2021++</th>
+      <th>Reunion 0.5 (March 2021)</th>
+      <th>Reunion 0.8 (Summer 2021)</th>
+      <th>Reunion 1.0 (Fall 2021)</th>
+      <th>2022 and Beyond</th>
     </tr>
     <tr>
-      <td>
+    <td>
+      <ul>
+            <li>1st supported public release</li>
+            <li>WinUI 3 to work in Desktop Packaged (Win32) Apps</li>
+            <li>Updates to MSIX to easily have their apps updates and repair</li>
+            <li>WinUI 3 and Win2D will be part of Reunion</li>
+       </ul>
+     </td>
+     <td>
         <ul>
-            <li>WinUI preview for UWP & Win32</li>
-            <li>Windows Virtual Desktop</li>
-            <li>WebView2 Preview</li>
-            <li>Project Reunion repo</li>
+            <li>Better bridging the gap between Win32 and WinRT</li>
+            <li>Delivery of Runtime dependencies for MSIX, Win32</li>
+            <li>State Notifications is supported</li>
+            <li>Framework Package Support</li>
+            <li>Enabling developers to install the runtime packages with their app</li> 
          </ul>
      </td>
-      <td>
+     <td>
         <ul>
-            <li>WinUI "go-live" Preview</li>
-            <li>Project Reunion Preview</li>
+            <li>Reunion Windowing release</li>
+            <li>New Rendering and Input features</li>
+            <li>Smooth WinUI input & composition system experiance</li>
+            <li>App Lifecycle features (Power Management, Activation)</li>   
          </ul>
      </td>
     <td>
         <ul>
             <li>Multiple releases per year</li>
             <li>Continue expanding existing app compatibility</li>
-            <li>Continue delivery of new capabilities</li>
+            <li>Continue delivery of new easier to use capabilities</li>
```

Comment:
Maybe change both of these "Continue" items to "Continuous" ?

Reply:
Or just eliminate "Continue" and simply have "Expanding app compatibility" and "More features" (while I love focusing on easier to use, for this high-level roadmap I don't think it's necessary).

Additionally we can tweak this after getting it updated initially, I'm okay with it as-is for this week